### PR TITLE
Make job history display handle pending jobs better.

### PR DESCRIPTION
**Pull request by Katharine**
/cc @stevekuznetsov

### CODE REVIEW

File: prow/cmd/deck/job_history.go [hunk 1]
```diff
@@ -302,13 +302,16 @@ func getBuildData(bucket storageBucket, dir string) (buildData, error) {
 	finished := gcs.Finished{}
 	err = readJSON(bucket, path.Join(dir, "finished.json"), &finished)
 	if err != nil {
+		b.Result = "Pending"
 		logrus.Infof("failed to read finished.json (job might be unfinished): %v", err)
```
Comment (stevekuznetsov) [inline prow/cmd/deck/job_history.go]: nit: could we move to a debug? or remove the line?

Comment (Katharine) [reply prow/cmd/deck/job_history.go]: debugified

**Comment by Katharine** [comment]
wow, this actually has tests.

**Comment by Katharine** [comment]
/hold

This doesn't actually pass tests. Not sure why we're suddenly retesting though.

**Comment by Katharine** [comment]
Fixed the tests on this, at last.

/hold cancel

**Pull request merged**